po/pl.po

```diff
@@ -852,24 +854,23 @@ msgid "Save File - %s"
 msgstr "Zapisz plik - %s"
 
 #: src/fileurlreader.cpp:28
-#, fuzzy, c-format
+#, c-format
 msgid "Error: failed to read URLs from file \"%s\": %s"
-msgstr "Błąd: Nie udało się wczytać odnośników z pliku \"%s\" (%s)"
+msgstr "Błąd: Nie udało się wczytać odnośników z pliku \"%s\" : %s"
 
 #: src/fileurlreader.cpp:66
-#, fuzzy, c-format
+#, c-format
 msgid "Error: failed to open file \"%s\": %s"
-msgstr "Błąd: Nie udało się otworzyć pliku \"%s\" (%s)"
+msgstr "Błąd: Nie udało się otworzyć pliku \"%s\": %s"
 
 #: src/filtercontainer.cpp:35 src/regexmanager.cpp:342 src/rssignores.cpp:42
 #, c-format
 msgid "couldn't parse filter expression `%s': %s"
 msgstr "błąd składniowy komendy filtra `%s': %s"
 
 #: src/formaction.cpp:145
-#, fuzzy
 msgid "usage: set <config-option> <value>"
-msgstr "użycie: set <zmienna>[=<wartość>]"
+msgstr "użycie: set <zmienna> <wartość>"
```

Comment:
Is "config option" really translated as "zmienna"? This message got fuzzy because there's another one below which says "set \<variable>[=\<value>]", and I suspect "zmienna" got here that way.

Reply:
Well it could be, but I changed it to better reflect the meaning. Nice catch.